src/utils.ts

```diff
@@ -1 +1,3 @@
 export const isServerSide = typeof document === 'undefined';
+
+export const ampScriptSrc = 'https://cdn.ampproject.org/v0.js';
```

Comment:
Another item to consider: AMP supports the `module`/`nomodule` pattern, and it might make sense to implement this `v0.mjs` etc.